src/shared-components/dropdown/desktopDropdown.js

```diff
@@ -88,6 +103,7 @@ DesktopDropdown.propTypes = {
   onOptionClick: PropTypes.func,
   onSelectClick: PropTypes.func,
   isOpen: PropTypes.bool,
+  maxHeight: PropTypes.string,
```

Comment:
It looks like this component is only consumed by `Dropdown`. Since we're providing `defaultProps` in that component, we should be able to make the height prop `isRequired` here.